src/services/mongoose/wallet-invoices.ts

```diff
@@ -91,14 +75,8 @@ export const WalletInvoicesRepository = (): IWalletInvoicesRepository => {
       return new RepositoryError(error)
     }
 
-    for await (const invoice of pending) {
-      yield {
-        paymentHash: invoice.id as PaymentHash,
-        walletId: invoice.walletId as WalletId,
-        selfGenerated: invoice.selfGenerated,
-        pubkey: invoice.pubkey as Pubkey,
-        paid: invoice.paid,
-      } as WalletInvoice
+    for await (const invoiceUser of pending) {
+      yield walletInvoiceFromRaw(invoiceUser)
```

Comment:
Perhaps we can converge to `...FromRecord` for these translation methods.

Reply:
should be done, but not really the goal of this PR